src/Model/Customer/CustomerBillingAddress.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+namespace Billogram\Model\Customer;
+
+/**
+ * @author Ibrahim Hizeoui <ibrahimhizeoui@gmail.com>
+ */
+class CustomerBillingAddress
+{
+    /**
+     * @var string
+     */
+    private $careOf;
+
+    /**
+     * @var bool
+     */
+    private $useCareOfAsAttention;
+
+    /**
+     * @var string
+     */
+    private $streetAddress;
+
+    /**
+     * @var string
+     */
+    private $zipCode;
+
+    /**
+     * @var string
+     */
+    private $city;
+
+    /**
+     * @var string
+     */
+    private $country;
+
+    /**
+     *
+     * @param string $careOf
+     * @param bool   $useCareOfAsAttention
+     * @param string $streetAddress
+     * @param string $zipCode
+     * @param string $city
+     * @param string $country
+     */
+    public function __construct(string $careOf, bool $useCareOfAsAttention, string $streetAddress, string $zipCode, string $city, string $country)
```

Comment:
What if I do not know all these values at the time I create a BillingAddress. 

Lets remove them from the constructor, just like we did with Customer.